### PR TITLE
FIX: Reset notebook kernel

### PR DESCRIPTION
**Pull request by kaitj**
Deterministic tractography solutions notebook was causing workflow to fail due to use of a local kernel. Reset it with this PR.

### CODE REVIEW

**Review by jhlegarreta — APPROVED** [review]
Have not tested.

**Comment by josephmje** [comment]
@kaitj could you try also getting rid of

```
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.9"
  }
 },
```

Just in case the python version listed there also interferes with the Github Actions

**Comment by kaitj** [comment]
> Just in case the python version listed there also interferes with the Github Actions

Done @josephmje (if this is what you meant).

File: code/deterministic_tractography/solutions/deterministic_tractography_solutions.ipynb [hunk 1]
```diff
@@ -516,22 +516,10 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "dipyvenv",
+   "display_name": "Python 3",
    "language": "python",
-   "name": "dipyvenv"
+   "name": "python3"
   },
```
Comment (josephmje) [inline code/deterministic_tractography/solutions/deterministic_tractography_solutions.ipynb:522]:
```suggestion
  }
```

**Comment by kaitj** [comment]
Should be fixed now.

**Review by josephmje — APPROVED** [review]

**Pull request merged**